plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -149,7 +150,14 @@ func findNodesThatFit(
 	nodeNameToInfo map[string]*schedulercache.NodeInfo,
 	nodes []*api.Node,
 	predicateFuncs map[string]algorithm.FitPredicate,
-	extenders []algorithm.SchedulerExtender) ([]*api.Node, FailedPredicateMap, error) {
+	extenders []algorithm.SchedulerExtender,
+	metaFactory interface{},
+) ([]*api.Node, FailedPredicateMap, error) {
+	// For the time being, the metadata factory is optional.  We may make it required at some point.
```

Comment:
Aah - ok. Let's leave as is (i.e. not throw an error), just clarify the comment that it's about the returned value, not the factory itself.